.github/scripts/publish-addon.sh

```diff
@@ -2,8 +2,7 @@
 set -e
 # this script publishes the addon zip file to the unofficial python2/3 kodi repo 
 
-VERSION=$1
```

Comment:
the version variable is still used in line 25.

Reply:
i think its not required to have it, you can also go to the zip file to get the referenced commit-hash.

Reply:
yeah but i mean probably you want to know which version relates to which commit hash...

Reply:
ah i got it wrong, because its the commit-hash of the other repo. the reference to the commit-hash doens't even matter. so i dont think we need the version in the commit message. maybe better remove it for simplification?